IbisHardwareIGSIO/ibishardwareIGSIO.cpp

```diff
@@ -392,38 +392,20 @@ void IbisHardwareIGSIO::InitPlugin()
 
     // Look for the Plus Toolkit path and Plus Server executable
     m_plusServerExec = GetIbisAPI()->GetCustomPath( PlusServerExecutable );
```

Comment:
m_plusServerExec should not be a member variable. The info is stored in the list of custom path. If you keep it here as well, you are duplicating information and run the risk of having inconsistent behaviour.